Translate the Chinese and give a short introduction to pubsub (#1497)
emqtt/emq-docs-cn: en_US/rule/bridge-gcp-pubsub.md

> # Bridge data to GCP Pub/Sub

Reviewer: ```suggestion
# Bridge data to GCP PubSub
```

> 

Reviewer: ```suggestion
Starting from EMQX Enterprise e4.4.11, EMQX's Rule-Engine supports Google GCP PubSub integration.

```

> EMQX GCP PubSub can send MQTT messages and client events to [Google Cloud Pub/Sub](https://cloud.google.com/pubsub?hl=zh-cn), which can help you flexibly choose services on Google Cloud and build IoT applications more easily.

Reviewer: ```suggestion
EMQX GCP PubSub integration can send MQTT messages and events to [Google Cloud PubSub](https://cloud.google.com/pubsub?hl=en-us), which can help you flexibly choose services on Google Cloud and build IoT applications more easily.
```
